### PR TITLE
[ELY-2333] Added forum links in README

### PR DESCRIPTION
**Pull request by fazer1929**
https://issues.redhat.com/browse/ELY-2333

### CODE REVIEW

**Review by skyllarr — APPROVED** [review]

**Comment by skyllarr** [comment]
Thank you @fazer1929 !

File: README.md [hunk 1]
```diff
@@ -38,3 +38,9 @@ All new features and enhancements should be submitted to 1.x branch only.
 Our [contribution guide](https://github.com/wildfly-security/wildfly-elytron/blob/1.x/CONTRIBUTING.md) will guide you through the steps for getting started on the WildFly Elytron project and will go through how to format and submit your first PR.
  
 For more details, check out our [getting started guide](https://wildfly-security.github.io/wildfly-elytron/getting-started-for-developers/) for developers.
+
+Get Help
+--------
+
+[Join the forum](https://groups.google.com/g/wildfly)\
+[Join Zulip Chat](https://wildfly.zulipchat.com/#narrow/stream/173102-wildfly-elytron)
```
Comment (fjuma) [inline README.md]: @fazer1929 Thanks for the PR! Just a minor comment, maybe we could add a few more details here, e.g., something like:

There are a couple ways to get in touch with us.

Feel free to ask questions on the WildFly user [forum](https://groups.google.com/g/wildfly)\.

The WildFly Elytron team also has an open chat room where you can listen in and ask questions. Join us on Zulip [chat](https://wildfly.zulipchat.com/#narrow/stream/173102-wildfly-elytron).

Comment (fazer1929) [reply README.md]: I have made the update

**Review by fjuma — APPROVED** [review]
Thanks @fazer1929!

**Pull request merged**